README.md

```diff
@@ -1,110 +1,197 @@
 
 To read an overview of how to migrate Mattermost to Kubernetes, [see this blog post](https://mattermost.com/blog/mattermost-kubernetes/).
 
-To install Mattermost Team Edition helm chart in GitLab helm chart deployment, [see this documentation](https://docs.mattermost.com/install/install-mmte-helm-gitlab-helm.html).
+The Mattermost Team Edition Helm chart can be found [here](https://github.com/helm/charts/tree/master/stable/mattermost-team-edition).
 
-# Pre requisites
+To install the Mattermost Team Edition Helm chart in a GitLab Helm chart deployment, [see this documentation](https://docs.mattermost.com/install/install-mmte-helm-gitlab-helm.html).
 
-To use the Mattermost Enterprise Helm Chart, you will first need a running Kubernetes cluster or Minikube.
+# 1. Prerequisites
 
-If you want to expose the application outside of your network, configure ingress in the Kubernetes cluster. Internally, Mattermost uses the [nginx-ingress](https://github.com/kubernetes/ingress-nginx). To install this in your Kubernetes cluster, follow [this documentation](https://kubernetes.github.io/ingress-nginx/deploy/). You may also use the [helm charts](https://github.com/helm/charts/tree/master/stable/nginx-ingress) directly.
+## 1.1 Kubernetes Cluster
 
-Moreover, if you want to automatically retrieve SSL certificates, use [cert-manager](https://github.com/jetstack/cert-manager). You also can use [kube-lego](https://github.com/jetstack/kube-lego), but it is deprecated. You can choose the one that you are most comfortable with. For cert-manager, you can follow [this documentation](https://cert-manager.readthedocs.io/en/latest/) and here is its [helm charts](https://github.com/helm/charts/tree/master/stable/cert-manager).
+You need a running Kubernetes cluster. If you do not have one, find options and installation instructions here:
 
-# Configuration
+https://kubernetes.io/docs/setup/pick-right-solution/ 
 
-To start, copy [mattermost-enterprise-edition/values.yaml](https://github.com/mattermost/mattermost-kubernetes/blob/master/mattermost-enterprise-edition/values.yaml) and name it `config.yaml`. This will be your configuration file for the Mattermost helm chart.
+## 1.2 Helm
 
-## DNS
+See: https://docs.helm.sh/using_helm/#quickstart
 
-Depending on the DNS service you're using, the exact steps will differ but generally to point your domain name at your release you can do the following:
+Once helm is installed and initialized, run the following:
 
-Note that your helm release must already be installed and running.
+```bash
+helm repo add mattermost https://releases.mattermost.com/helm
+helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+```
```

Comment:
```suggestion
```

We recommend installing Helm v2.13.1 or later.
```

Reply:
I don't think that's the correct change is it?

Reply:
Oh GitHub is screwing it up because of the extra ticks. I see the change and will make it manually